03.image_classification/resnet.py

```diff
@@ -36,26 +36,25 @@ def conv_bn_layer(input,
     return paddle.layer.batch_norm(input=tmp, act=active_type)
 
 
-def shortcut(ipt, n_in, n_out, stride):
-    if n_in != n_out:
-        return conv_bn_layer(ipt, n_out, 1, stride, 0,
+def shortcut(ipt, ch_in, ch_out, stride):
+    if ch_in != ch_out:
+        return conv_bn_layer(ipt, ch_out, 1, stride, 0,
                              paddle.activation.Linear())
     else:
         return ipt
 
 
-def basicblock(ipt, ch_out, stride):
-    ch_in = ch_out * 2
```

Comment:
The original writing before v2 API is as follows,
```
ch_in = ipt.num_filters
```
The `num_filters` can be got after merging [PR](https://github.com/PaddlePaddle/Paddle/pull/2288).